examples/ads.amp.html

```diff
@@ -247,6 +248,30 @@ <h2 id="adgeneration">Ad Generation</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="adhese">Adhese</h2>
+  <amp-ad width=300 height=250
+      type="adhese"
+      data-location="_sdk_amp_"
+      data-position=""
```

Comment:
data-location is indeed required and should always be filled in.
It is data-position that can be empty which is the case here.